src/core_tests.rs

```diff
@@ -43,6 +43,16 @@ use xor_name::XorName;
 // Poll one event per node. Otherwise, all events in a single node are polled before moving on.
 const BALANCED_POLLING: bool = true;
 
+// Toggle iterations for full or quick test according to env_var
+fn iterations() -> usize {
```

Comment:
Quite generic a name to be tied to one specific test. Why not inline in the test or make it return a bool ("do full test")?